daphne/src/lib.rs

```diff
@@ -496,6 +626,7 @@ pub struct DapLeaderUncommitted {
 /// The Helper's state during the aggregation flow.
 #[derive(Clone, Debug, PartialEq)]
 pub struct DapHelperState {
+    pub(crate) part_batch_sel: PartialBatchSelector,
```

Comment:
Reviewer note: This structure carries state across aggregate requests. The Helper needs to be able to remember the batch ID indicated by the Leader in its AggregateInitializeReq.